test_utils/model_whisper_utils.py

```
@@ -0,0 +1,75 @@
+            word_timestamps=True,
+            multilingual=False,
+            language="en",
+        )
+        for segment in segments:
+            for word in segment.words:
+                #print("[%.4fs -> %.4fs] %s" % (word.start, word.end, word.word))
+                words.append(word)
+            #print("[%.2fs -> %.2fs] %s" % (segment.start, segment.end, segment.text))
+            texts.append(segment.text.strip())
+
+        transcription = ' '.join(texts).strip()
```

Ruff (0.12.2)

Local variable `transcription` is assigned to but never used

Remove assignment to unused variable `transcription`

(F841)